magicctapipe/scripts/lst1_magic/tune_magic_nsb.py

```diff
@@ -0,0 +1,334 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# script to tune NSB level of MAGIC part of sim_telarray MCs
+# inspired by calculate_noise_parameters in lstchain/image/modifier.py
+# but adapted for the specifics of the MAGIC data calibration
+
+"""
+Usage:
+
+$> python tune_magic_nsb.py
+--config  config_file.json     (MCP generic config file to take extractor parameters)
+--input-mc  simtel_file.simtel.gz    simulation simtel file (for the same period as the data!)
+--input-data 2..._Y_.root          real data DL1 file
+
+Calculates the parameters needed to tune the NSB in MC DL1 files
+to the level of NSB in a given data file
+"""
+
+import argparse
+import json
+import logging
+import sys
+from pathlib import Path
+
+import numpy as np
+import yaml
+from ctapipe.calib.camera import CameraCalibrator
+from ctapipe.io import EventSource
+from ctapipe_io_magic import MAGICEventSource
+from scipy.interpolate import interp1d
+from traitlets.config import Config
+
+log = logging.getLogger(__name__)
+
+parser = argparse.ArgumentParser(description="Tune MAGIC NSB")
```

Comment:
Maybe the parser could be put in the main (instead of using global variables), but not a real issue (like this it is fine and it runs)